Tools/GNUMake/Make.machines

```diff
@@ -9,7 +9,7 @@ ifdef $(HOSTNAME)
 else ifdef $(HOST)
   host_name := $(strip $(HOST))
 else
-  host_name := $(shell hostname -f)
+  host_name := $(shell hostname)
```

Comment:
This might break on other machines that include `Make.$(host_name)`.  Maybe we can keep the way `host_name` is and add an extra variable `short_host_name := $(shell hostname -s)`.